### PR TITLE
Add skip-tls-verify option

### PR DESCRIPTION
**Pull request by EinAeffchen**
This pull request solves issue #114

### CODE REVIEW

**Comment by jpmckinney** [comment]
It looks like you ran a linter. Can you submit a PR that only makes the necessary changes?

**Comment by EinAeffchen** [comment]
I've removed the linting and only changed the necessary code, I haven't updated the version file yet though.

File: scrapyd_client/deploy.py [hunk 1]
```diff
@@ -213,48 +214,40 @@ def _upload_egg(target, eggpath, project, version):
...
     _log('Deploying to project "%s" in %s' % (project, url))
-    return _http_post(req)
+    res = requests.post(
+        url, data=body, headers=headers, verify=not skip_tls_verify
+    )
+    if res.ok:
+        _log("Server response (%s):" % res.status_code)
+    else:
+        _log("Deploy failed (%s):" % res.status_code)
+    _log(json.dumps(res.json(), indent=3))
```
Comment (jpmckinney) [inline scrapyd_client/deploy.py:234]: This code is not equivalent to `_http_post`, which `print()`'ed successful responses without indentation, didn't print anything on `URLError` and tried to do more on `HTTPError`.

I recommend just keeping `_http_post` as-is and making this a much more minimal PR.

**Comment by jpmckinney** [comment]
Closing as abandoned.

**Pull request closed without merging**